server/src/game/event/on_phase_start.rs

```diff
@@ -0,0 +1,24 @@
+use crate::game::{phase::PhaseType, player::PlayerReference, Game};
+
+#[must_use = "Event must be invoked"]
+pub struct OnPhaseStart{
+    pub phase: PhaseType
```

Comment:
Why public? There's a `new` function you can use

Reply:
fixed